libs/coin-modules/coin-evm/package.json

```diff
@@ -85,6 +85,7 @@
   "devDependencies": {
     "@ledgerhq/disable-network-setup": "workspace:^",
     "@ledgerhq/hw-app-eth": "workspace:^",
+    "@ledgerhq/live-config": "workspace:^",
```

Comment:
Well it won't impact us much with the apps and the way it's bundled but we should probably have this in the `dependencies` object and not the dev one

Also I don't think we should add this dep in the coin modules, usually if you check the config file in coin-solana for example we pass in the data from common